scripting/get5/backups.sp

```diff
@@ -292,11 +300,13 @@ public bool RestoreFromBackup(const char[] path) {
     kv.GoBack();
   }
 
+  int roundNumberRestoredTo = -1; // Assume -1 for non-live backup
   char tempValveBackup[PLATFORM_MAX_PATH];
   GetTempFilePath(tempValveBackup, sizeof(tempValveBackup), TEMP_VALVE_BACKUP_PATTERN);
   if (kv.JumpToKey("valve_backup")) {
     g_SavedValveBackup = true;
     kv.ExportToFile(tempValveBackup);
+    roundNumberRestoredTo = kv.GetNum("round", 0) - 1; // Round number is not zero-indexed inside valve backup
```

Comment:
Need to verify that this is correct; that the round number actually matches the one we restored to.